src/pages/profile/help/index.tsx

```diff
@@ -1,22 +1,94 @@
-import React from 'react'
+import React, { useEffect } from 'react'
+import { Controller, useForm } from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+import { IHelpDataProps } from '@allTypes/reduxTypes/supportStateTypes'
+import { Translation } from '@constants/translations'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { dispatch, useAppSelector } from '@redux/hooks'
+import { supportMiddleware, supportSelector } from '@redux/slices/support'
 import { Button } from '@uiComponents/Button'
 import { Input } from '@uiComponents/Input'
+import { helpValidationSchema } from '@validation/support/help'
 
-export const Help = () => (
-  <div className="grid gap-4 rounded bg-gray-thin p-5 xl:p-10">
-    <h2 className="text-lg">Tell us about your problem:</h2>
-    <p className="text-base text-black-light">
-      Try to support your explanation with a screenshot, with that we can solve your problem faster.
-    </p>
-    <Input placeholder="Headline" />
-    <Input
-      placeholder="Type your problem here..."
-      color="primary"
-      rows={7}
-    />
-    <Button>Submit</Button>
-  </div>
-)
+const defaultValues = {
+  headline: '',
+  problem: '',
+}
+
+export const Help = () => {
+  const [t] = useTranslation()
+  const { helpStatus } = useAppSelector(supportSelector.support)
+
+  const { control, handleSubmit, reset } = useForm({
+    defaultValues,
+    mode: 'onSubmit',
+    resolver: yupResolver(helpValidationSchema),
+  })
+
+  const onSubmit = (data: IHelpDataProps) => {
+    dispatch(
+      supportMiddleware.fetchHelpMessage({
+        headline: data.headline,
+        problem: data.problem,
+      })
+    )
+  }
+
+  useEffect(() => {
+    if (helpStatus === 200) {
+      reset({ ...defaultValues })
+    }
+  }, [helpStatus, reset])
+
+  return (
+    <div className="grid gap-4 rounded bg-gray-thin p-5 xl:p-10">
+      <h2 className="text-lg">{t(Translation.HELP_TEL_PROBLEM)}</h2>
+      <p className="text-base text-black-light">{t(Translation.HELP_TEXT)}</p>
+      <form
+        onSubmit={handleSubmit(onSubmit)}
+        className="w-full text-center"
+      >
+        <div className="mb-5">
+          <Controller
+            control={control}
+            name="headline"
+            defaultValue=""
+            render={({ field, fieldState }) => (
+              <Input
+                {...field}
+                placeholder={`${t(Translation.HELP_HEADLINE)}`}
+                error={fieldState.error ? t(Translation.HELP_HEADLINE_REQUIRED) : null}
+              />
+            )}
+          />
+        </div>
+        <div className="mb-5">
+          <Controller
```

Comment:
Use hook`useController` instead of Controller wrapper

Reply:
done